meson.build

```diff
@@ -79,7 +79,6 @@ test_args = [
   '-Wmissing-prototypes',
   '-Wnested-externs',
   '-Wno-error=cpp',
-  '-Wno-discarded-qualifiers',
```

Comment:
Maybe explicitly switch this to `-Wdiscarded-qualifiers` to make it clear that it’s a warning we care about?